packages/math-input/src/components/keypad/button-assets.tsx

```diff
@@ -341,26 +272,40 @@ export default function ButtonAsset({id}: Props): React.ReactElement {
                     />
                 </svg>
             );
+        // TODO(ned): Per the notes in `KeyConfigs`, shouldn't this be a comma
+        // that we replace with the period icon for i18n? Duplicating for now.
         case "DECIMAL":
             return (
                 <svg
-                    width="40"
-                    height="40"
-                    viewBox="0 0 40 40"
-                    fill="none"
                     xmlns="http://www.w3.org/2000/svg"
+                    width="32"
+                    height="32"
+                    fill="currentColor"
+                    viewBox="0 0 256 256"
+                >
+                    <path
+                        d="M140,128a12,12,0,1,1-12-12A12,12,0,0,1,140,128Z"
+                        // moves decimal down to baseline of number icons,
+                        // otherwise indistinguishable from cdot
+                        transform="translate(0 80)"
+                    />
+                </svg>
+            );
+        case "PERIOD":
```

Comment:
```suggestion
        case "DECIMAL":
        case "PERIOD":
```

Instead of duplicating it, would it make sense to share a case? I realize they're going to (hopefully) diverge soon, so not a big deal either way.